src/Panel.vue

```diff
@@ -1,27 +1,25 @@
 <template>
-    <span class="panel-container">
+    <span class="card-container">
         <div class="morph" v-show="localMinimized">
-            <div class="morph-display-wrapper" @click="open()">
-                <button class="morph-display-button btn btn-default">
-                    <template v-if="altContent">
-                        <div class="panel-title" v-html="altContent"></div>
-                    </template>
-                    <template v-else>
-                        <slot name="header">
-                            <div class="panel-title" v-html="altContent"></div>
-                        </slot>
-                    </template>
-                </button>
-            </div>
+            <button :class="['morph-display-wrapper', 'btn', btnType, 'card-title']" @click="open()">
+                <template v-if="altContent">
+                    <div  v-html="altContent"></div>
+                </template>
+                <template v-else>
+                    <slot name="header">
+                        <div v-html="altContent"></div>
+                    </slot>
+                </template>
+            </button>
         </div>
-        <div :class="['panel', panelType, {'expandable-panel': isExpandablePanel}]" v-show="!localMinimized">
-            <div :class="['panel-heading',{'accordion-toggle':canCollapse}]"
+        <div :class="['card', {'expandable-card': isExpandableCard}, borderType]" v-show="!localMinimized">
+            <div :class="['card-header',{'accordion-toggle':canCollapse}, cardType, borderType]"
                  @click.prevent.stop="canCollapse && toggle()"
                  @mouseover="onHeaderHover = true" @mouseleave="onHeaderHover = false">
                 <div class="header-wrapper">
                     <span :class="['caret', {'caret-collapse': !localExpanded}]" v-show="showCaret"></span>
```

Comment:
Will this be eventually be replaced by a glyphicon? Or is there a unicode equivalent that we can use for the caret?

Reply:
Update: While porting Dropdown, I saw this in the migration guide:

> Dropdown toggles no longer require an explicit `<span class="caret"></span>;` this is now provided automatically via CSS’s `::after` on `.dropdown-toggle`.

I checked the CSS, but couldn't find out how they render the caret through the `::after` selector. The content is `""` yet it seems to render the caret somehow? Seems kinda magical 🤔. Will let you know if I found anything new.